dotnet-setversion.sln

```diff
@@ -22,6 +22,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "test-netcore-2", "test\test
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "test-issue-8", "test\test-issue-8\test-issue-8.csproj", "{32FC873A-C2C8-44D4-AB7B-A108BA0CB3BA}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "integration", "test\integration\integration.csproj", "{FEE152BC-1F28-4BAE-98F1-E838622FF6CF}"
```

Comment:
Adding this project under `test/` will cause the end-to-end scripts to break, but that shouldn't be too hard to resolve.

Reply:
Although maybe with the integration tests, I can remove the e2e testing anyway. The integration tests you added are probably sufficient.

Reply:
I'm not familiar with how AppVeyor works. Can I leave fixing this up to you?

Reply:
Yeah sure.